src/cryptoadvance/specter/translations/.gitignore

```diff
@@ -0,0 +1 @@
+./**/messages.mo
```

Comment:
Why is this new file needed? Should just be `messages.mo` in the existing .gitignore I think

Reply:
I think it's a styling-question. Putting everything in one .gitignore is possible but maintaining gets a bit more difficult with bigger projects.
I moved it to the central one but kept a more specific path. I think that's a good idea in order to get a better idea why (where) certain files should be ignored.